ga4gh/datamodel/variants.py

```diff
@@ -238,6 +244,35 @@ def getVariants(self, referenceName, startPosition, endPosition,
                 else:
                     break
 
+    def getCallSets(self, name, startCallSetPosition):
+        """
+        Returns an iterator over the specified callset name. The parameters
+        correspond to the attributes of a GASearchCallSetsReuqest object.
+        """
+        assert name is not None
+        callSetIds = self._sampleNames[startCallSetPosition:]
+        # get all substrings of name in a list nameStrings
+        l = len(name)
```

Comment:
we don't use single-letter variable names, so change this to something like nameLength